tests/e2e/specs/keplr/keplr-spec.js

```diff
@@ -75,6 +75,16 @@ describe('Keplr', () => {
         expect(setupFinished).to.be.true;
       });
     });
+
+    it(`should get wallet address while running addNewTokensFound flow`, () => {
+      cy.getWalletAddress('Agoric localhost').then(walletAddress => {
+        expect(walletAddress.length).to.be.equal(45);
+      });
+
+      cy.getWalletAddress('Cosmos Hub').then(walletAddress => {
+        expect(walletAddress.length).to.be.equal(45);
+      });
+    });
```

Comment:
@rabi-siddique  we have two sets of these test cases. both which don't technically run the addNewTokensFound flow. we should fix this in a later PR

Reply:
@frazarshad The test cases you're referring to are working fine. `addNewTokensFound` flow is essential for reliable results.